mlflow/types/agent.py

```diff
@@ -55,7 +50,7 @@ class ChatAgentMessage(BaseModel):
 
     if IS_PYDANTIC_V2_OR_NEWER:
 
-        @model_validator(mode="after")
+        @model_validator
```

Comment:
this will fail when used, since the model_validator does not contain a mode, but then the decorator actually passes a mode... so decorator will not be able to instanciate the function

Reply:
Thanks, good catch! I'm having a hard time checking my work, because the unit tests all pass locally but I'm not confident that the tests I'm running are really checking everything. Sorry for all the commits 🙈